pkg/url/url.go

```diff
@@ -45,7 +45,7 @@ func Delete(client *occlient.Client, urlName string, applicationName string) err
 // Create creates a URL and returns url string and error if any
 // portNumber is the target port number for the route and is -1 in case no port number is specified in which case it is automatically detected for components which expose only one service port)
 func Create(client *occlient.Client, urlName string, portNumber int, componentName, applicationName string) (string, error) {
-	labels := urlLabels.GetLabels(urlName, componentName, applicationName, false)
+	labels := urlLabels.GetLabels(urlName, componentName, applicationName, true)
```

Comment:
This is important as testing actually FOUND a bug! We never added any additional labels to URL creation... Now we do.